src/code/z_shrink_window.c

```diff
@@ -5,39 +5,46 @@
 #include "prevent_bss_reordering.h"
 #include "global.h"
 
+typedef struct {
+    /* 0x0 */ s8 letterboxTarget;
+    /* 0x1 */ s8 letterboxSize;
+    /* 0x2 */ s8 pillarboxTarget;
+    /* 0x3 */ s8 pillarboxSize;
+} ShrinkWindowContext; // size = 0x4
```

Comment:
```suggestion
} ShrinkWindow; // size = 0x4
```
I think we need to reassess what structs should be given the name `*Context`. In this spot I find it completely unnecessary given the struct's limited scope and brevity of the type.